api/hydrophoneApi.go

```diff
@@ -77,20 +82,20 @@ func InitApi(
 	ntf clients.Notifier,
 	sl shoreline.Client,
 	gatekeeper commonClients.Gatekeeper,
-	metrics highwater.Client,
 	seagull commonClients.Seagull,
 	templates models.Templates,
 ) *Api {
+	logger := log.New(os.Stdout, CONFIRM_API_PREFIX, log.LstdFlags|log.Lshortfile)
```

Comment:
Why creating the logger with log.LstdFlags|log.Lshortfile flags?
It will always show the same line number (332)